### PR TITLE
Update circle.yml

### PR DESCRIPTION
**Pull request by kkowalsky**
"tickler" is now called beacon.

### CODE REVIEW

**Comment by nvkelso** [comment]
Why doesn't Giphy have a good Tickle Me Elmo GIF?!

**Comment by kkowalsky** [comment]
NO @nvkelso. THIS IS WHY THERE WAS A RENAME.

File: circle.yml [hunk 1]
```diff
@@ -27,10 +27,10 @@ deployment:
   prerelease:
     branch: "master"
     commands:
-      - pip install 'Circle-Tickler == 1.0.1'
```
Comment (nvkelso) [inline circle.yml]: There's a smart quote here, should probably be a single basic quote mark.

File: circle.yml [hunk 2]
```diff
@@ -27,10 +27,10 @@ deployment:
...
-      - tickle-circle mapzen documentation tilezen-buildbot/tiles-prerelease-docs $CIRCLE_TOKEN
+      - pip install ‘Circle-Beacon == 2.0.0'
+      - alert-circle mapzen documentation master $CIRCLE_TOKEN
   release:
     tag: /.+/
     commands:
-      - pip install 'Circle-Tickler == 1.0.1'
```
Comment (nvkelso) [inline circle.yml]: Ditto on smart quote > basic quote

**Comment by kkowalsky** [comment]
oh no, thanks for the catch @nvkelso 

mondays amirite?

**Comment by kkowalsky** [comment]
just fixed.

**Pull request merged**